Doc/library/sys.monitoring.rst

```diff
@@ -284,6 +284,11 @@ performance monitoring. For example, a program can be run under a
 debugger with no overhead if the debugger disables all monitoring
 except for a few breakpoints.
 
+.. function:: restart_events()
+
+   Enable all the events that were disabled by :data:`sys.monitoring.DISABLE`
+   for all tools.
+
```

Comment:
oh no, I'll have to update the comment at https://github.com/python/typeshed/blob/5c775a3502af9264c74c69ebc1841a95b0147e04/stdlib/sys/_monitoring.pyi#L47 😆

In all seriousness, was this just an omission from the original PR that added documentation for `sys.monitoring`? You know more than me -- I was wondering whether it was maybe _meant_ to be an undocumented implementation detail?

Reply:
I believe @markshannon just forgot to add this. `DISABLE` and `MISSING` were not in the initial docs either. I added them in when I polished the docs. The function is mentioned in PEP 669 and it's just like other functions.

Feature wise, we should have a public function to reset the disables from the users - or it would be kind confusing.